.github/ISSUE_TEMPLATE/bug_report_form.yml

```
@@ -0,0 +1,147 @@
+name: "Calendar Bug"
+description: "Have you encountered a bug?"
+labels: ["bug", "0. to triage"]
+body:
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Describe the steps to reproduce the bug.
+        The better your description is _(go 'here', click 'there'...)_ the fastest you'll get an _(accurate)_ answer.
```

I'm just being pedant
```suggestion
        The better your description is _(go 'here', click 'there'…)_ the fastest you'll get an _(accurate)_ answer.
```